data/subprocesses-output.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @title Subprocesses: Collecting Output
+ * @difficulty intermediate
+ * @tags cli
+ * @run --allow-run <url>
+ * @resource {https://deno.land/api?s=Deno.Command} Doc: Deno.Command
+ *
+ * We don't often write programs in isolation. In a lot of cases we want
+ * to interact with the outside system and the shell is a common way to do this.
+ */
+
+// The Deno namespace has a unified api for interacting with the shell called
+// Deno.command. With it, we can initialize some information about the command
+// but it will not be executed immediately.
+const command = new Deno.Command("deno", {
+  args: ["eval", "console.log('hello from deno')"],
```

Comment:
How about also adding something like `console.error('hello in stderr')` in eval script to better illustrate `console.log("stderr:", textDecoder.decode(result.stderr))` part?